test/jobs/template_render_job_test.rb

```diff
@@ -21,11 +21,11 @@ def render_job(composer_params)
     it 'render report and delivers it to mail' do
       mailer = mock('mailer')
       mailer.expects('deliver_now')
+      composer_params = { 'foo' => 'bar', 'send_mail' => true, 'mail_to' => 'this@email.cz', 'gzip' => true }
```

Comment:
While you're at it: please also use a domain like example.com (https://community.theforeman.org/t/using-ips-and-domains-in-tests-documentation/16600)

Reply:
Done that